dod/src/test/java/com/revolut/rxdata/dod/DataObservableDelegateTest.kt

```diff
@@ -803,4 +804,32 @@ class DataObservableDelegateTest {
             .assertValueCount(8)
     }
 
+    @Test
+    fun `WHEN network request is over THEN disposable is removed from container`() {
+        val disposableContainer = CompositeDisposable()
+        val dataObservableDelegate = DataObservableDelegate(
+            fromNetwork = fromNetworkScoped,
+            fromMemory = fromMemory,
+            toMemory = toMemory,
+            fromStorage = fromStorage,
+            toStorage = toStorage,
+            networkSubscriptionsContainer = disposableContainer
+        )
+
+        whenever(fromNetwork.invoke(eq(params))).thenReturn(Single.fromCallable { domain })
+
+        val testObserver =
+            dataObservableDelegate.observe(params = params, forceReload = true).test()
+
+        testObserver.assertValueCount(1)
+        testObserver.assertValueAt(0, Data(null, error = null, loading = true))
+
+        ioScheduler.advanceTimeBy(10, TimeUnit.MILLISECONDS)
+        testObserver.assertValueCount(2)
+
+        testObserver.assertValueAt(1, Data(domain, error = null, loading = false))
```

Comment:
pls remove unrelated assertions for readability, you can do 
`observable.test().takeUntilLoaded().apply { ioScheduler.triggerActions() }.assertComplete()` and then assert disposableContainer size right away

Reply:
FYI there's a PR just merged that update testing framework, don't forget to merge master in just in case, although it doesn't seem like there's gonna be any problems

Reply:
done, thanks